src/store/item.ts

```diff
@@ -51,16 +51,22 @@ export const useItemStore = defineStore("item", {
 
     async saveData(podcastname: string, docname: string) {
       const docRef = doc(collection(db, podcastname), docname)
-      return setDoc(docRef, {
-        items: this.itemList,
-        slotTitles: this.slotTitleList,
-      })
+      // eslint-disable-next-line no-useless-catch
+      try {
+        setDoc(docRef, {
+          items: this.itemList,
+          slotTitles: this.slotTitleList,
+        })
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } catch (e: any) {
+        throw e
+      }
```

Comment:
not sure why you need the try/catch since you're not doing anything in the catch